statsmodels/miscmodels/robust_genericmle.py

```diff
@@ -0,0 +1,418 @@
+# -*- coding: utf-8 -*-
+"""
+
+Created on Thu Feb 27 13:32:36 2014
+
+Author: Josef Perktold
+"""
+
+import numpy as np
+from statsmodels.base.model import GenericLikelihoodModel
+from statsmodels.sandbox.regression import gmm
+
+import statsmodels.robust.norms as rnorms
+import statsmodels.robust.scale as rscale
+import statsmodels.robust.robust_linear_model as rlm
+from nltk.corpus.reader.rte import norm
+
+
+class MEstimator(GenericLikelihoodModel):
+
+    def __init__(self, *args, **kwds):
+        self.norm = rnorms.HuberT()
+        super(MEstimator, self).__init__(*args, **kwds)
+        self.scale_fixed = None
+
+    def transform_params(self, params):
+
+        if self.scale_fixed is None:
+            params_loc, scale = params[:-1], params[-1]
+            scale = np.abs(scale)
+        else:
+            params_loc = params
+            scale = self.scale_fixed
+
+        return params_loc, scale
+
+
+    def nloglikeobs(self, params):
+        # Note: norms is a loss function, so this is nloglikeobs
+
+        params_loc, scale = self.transform_params(params)
+
+        #nobs = len(self.endog)
+        resid_scaled = (self.endog - np.dot(self.exog, params_loc)) / scale
+        scale_fac = 0.491
+        rho_fac = scale * scale_fac
+        objective = (rho_fac * self.norm(resid_scaled / scale_fac) +
+                      rho_fac * np.log(scale * scale_fac))
```

Comment:
what is rho_fac?

this looks like analogue to mle, with log(scale)
shouldn't it have scale_bias as factor in second term  `scale_bias * log(scale)`
scale_bias should not be in the first term `norm(...)`